helpers/component_info.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
no changes needed in `component_info.json`. Just move it to `helpers` directory unchanged.

Reply:
Okay so like I should not make any changes in the error codes right?

Reply:
yes